src/scenes/Engagement/InternshipEngagement/InternshipEngagementDetail.tsx

```
@@ -186,6 +186,18 @@ export const InternshipEngagementDetail = ({ engagement }: EngagementQuery) => {
                     onClick={() => show('countryOfOriginId')}
                   />
                 </Grid>
+                <Grid item>
+                  <DataButton
+                    onClick={() => show(['webId'])}
+                    secured={engagement.webId}
+                    redacted="You do not have permission to view Web ID"
+                    children={
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)